frontend/public/index.html

```diff
@@ -30,24 +30,24 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <!-- Page title -->
-    <title>Junction App</title>
+    <title>%REACT_APP_SEO_PAGE_TITLE%</title>
 
     <!-- SEO Meta tags -->
-    <meta name="title" content="Experience Europe’s Leading Hackathon" />
-    <meta property="og:title" content="Experience Europe’s Leading Hackathon" />
-    <meta name="twitter:title" content="Experience Europe's Leading Hackathon" />    
-    <meta name="description" content="Organized in the Helsinki Area, Finland, Junction is a meeting place for thousands of developers, designers, and entrepreneurs. A weekend-long experience, gathering tech enthusiasts from all over the world to create with the latest technology in a unique environment and atmosphere."/>
-    <meta property="og:description" content="Organized in the Helsinki Area, Finland, Junction is a meeting place for thousands of developers, designers, and entrepreneurs. A weekend-long experience, gathering tech enthusiasts from all over the world to create with the latest technology in a unique environment and atmosphere." />
-    <meta name="twitter:description" content="Organized in the Helsinki Area, Finland, Junction is a meeting place for thousands of developers, designers, and entrepreneurs. A weekend-long experience, gathering tech enthusiasts from all over the world to create with the latest technology in a unique environment and atmosphere." />
+    <meta name="title" content="%REACT_APP_SEO_PAGE_TITLE%" />
+    <meta property="og:title" content="%REACT_APP_SEO_PAGE_TITLE%" />
+    <meta name="twitter:title" content="%REACT_APP_SEO_PAGE_TITLE%" />    
+    <meta name="description" content="%REACT_APP_SEO_PAGE_DESCRIPTION%"/>
+    <meta property="og:description" content="%REACT_APP_SEO_PAGE_DESCRIPTION%" />
+    <meta name="twitter:description" content="%REACT_APP_SEO_PAGE_DESCRIPTION%" />
     
     <meta name="og:type" content="website" />
-    <meta property="og:image" content="https://res.cloudinary.com/hackjunction/image/upload/c_fill,g_center,h_630,w_1200/rqn6uonv2nuzmkyanrqj"/>
-    <meta name="twitter:image" content="https://res.cloudinary.com/hackjunction/image/upload/c_fill,g_center,h_600,w_1200/rqn6uonv2nuzmkyanrqj" />
+    <meta property="og:image" content="%REACT_APP_SEO_IMAGE_URL%"/>
+    <meta name="twitter:image" content="%REACT_APP_SEO_IMAGE_URL%" />
     <meta property="og:image:width" content="1200"/>
     <meta property="og:image:height" content="630"/>
     <meta name="twitter:card" content="summary_large_image">
-    <meta name="twitter:site" content="@hackJunction">
-    <meta name="twitter:creator" content="@hackJunction">
+    <meta name="twitter:site" content="%REACT_APP_SEO_TWITTER_HANDLE%">
+    <meta name="twitter:creator" content="%REACT_APP_SEO_TWITTER_HANDLE%">
 
```

Comment:
So for this to work, we'll need to make the following additions to the `.env` for all deployments:

```
REACT_APP_SEO_PAGE_TITLE=Experience Europe’s Leading Hackathon
REACT_APP_SEO_PAGE_DESCRIPTION=Organized in the Helsinki Area, Finland, Junction is a meeting place for thousands of developers, designers, and entrepreneurs. A weekend-long experience, gathering tech enthusiasts from all over the world to create with the latest technology in a unique environment and atmosphere.
REACT_APP_SEO_IMAGE_URL=https://res.cloudinary.com/hackjunction/image/upload/c_fill,g_center,h_630,w_1200/rqn6uonv2nuzmkyanrqj
REACT_APP_SEO_TWITTER_HANDLE=@hackJunction
```

Reply:
@Earlo can you take care of this? Once it's done I would say this one's safe to merge

Reply:
I see no problems with this